addons/compat_3as/CfgVehicles.hpp

```diff
@@ -0,0 +1,47 @@
+class CfgVehicles {
+    class Helicopter_Base_H;
+    class 3as_laat_Base: Helicopter_Base_H {
+        GVARMAIN(passengerTurrets)[] = {
+            // Left and right ramp seats, side seats, inside standing seats
+            {1}, {2},
+            {3}, {4},
+            {5}, {6},
+            {7}, {8},
+            {9}, {10},
+            {11}, {12},
+            {13}, {15},
+            {16}, {17},
+            {18}, {19},
+            {20}, {21},
+            {22}, {23},
+            {24}, {25},
+            {26}, {27},
+            {28}, {29},
+            {30}, {31},
+            {32}, {33},
+            {34}
+        };

```

Comment:
Don't worry about these, I made some changes (that should've already been present) to remove the need for the passenger turrets

Reply:
Unless 3AS has FFV positions that aren't marked as `showAsCargo = 1`, in which case they would need to be added.